src/components/NavBar/ChainSwitcher.tsx

```diff
@@ -83,7 +83,7 @@ export const ChainSwitcher = ({ leftAlign }: ChainSwitcherProps) => {
     <NavDropdown top={54} leftAligned={leftAlign} paddingBottom={8} paddingTop={8}>
       <Column marginX="8">
         {NETWORK_SELECTOR_CHAINS.map((chainId: SupportedChainId) =>
-          isSupported ? (
+          isChainAllowed(chainId) ? (
```

Comment:
is this necessary? chains here should always be supported

Reply:
agreed

Reply:
removed